css/amp.css

```diff
@@ -617,6 +617,13 @@ form [submit-error] {
   display: none;
 }
 
+/**
+ * Form validation error messages should be hidden at first.
+ */
+span[visible-when-invalid] {
```

Comment:
I don't believe there is a requirement on span,  just `[visible-when-invalid]` should do.